force-app/main/adapter/in/sobjects/account/AccountAdapter.cls

```diff
@@ -100,6 +100,24 @@ public inherited sharing class AccountAdapter extends fflib_SObjects2 {
             setCustomNamingField();
             setNameAndGreetingsToReplacementText();
         }
+        revertAllMembersDeceased();
```

Comment:
Since this only runs when the flag is **not** set, I'd either move the `if (isAllMembersDeceasedUpdateEnabled)` block up here (probably bad from a mixed logic perspective) or rename it something like `revertAllMembersDeceasedIfUpdateNotEnabled`